From 43a86ab9ab37fa05dcc443544645b3ea07603a89 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Sun, 14 Dec 2025 23:25:51 -0700
Subject: [PATCH] fix: Use argparse format

* pyslha used to use the legacy optparse format, which diverges from
  argparse's format in Python 3.7+. Use '%(prog)s' to avoid

  > ValueError: unsupported format character 'p' (0x70) at index 8
---
 bin/isawig2slha | 2 +-
 bin/slha2isawig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/isawig2slha b/bin/isawig2slha
index 1a34608..a6ee4ca 100755
--- a/bin/isawig2slha
+++ b/bin/isawig2slha
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 
 """\
-Usage: %prog IN.isa [OUT.spc]
+Usage: %(prog)s IN.isa [OUT.spc]
 
 Convert a HERWIG/ISAWIG model/decay spectrum file to an SLHA spectrum input
 file.
diff --git a/bin/slha2isawig b/bin/slha2isawig
index a8dc4f6..cde8cd7 100755
--- a/bin/slha2isawig
+++ b/bin/slha2isawig
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 
 """\
-Usage: %prog IN.slha [OUT.isa]
+Usage: %(prog)s IN.slha [OUT.isa]
 
 Convert an SLHA model/decay spectrum file to a HERWIG/ISAWIG SUSY spectrum input
 file, in the format written by ISAWIG. The intention of this script is to allow
-- 
2.52.0

